### PR TITLE
change Destinatario into persona fisica and add feature file

### PR DESCRIPTION
**Pull request by delfinoroNTTDATA**
add 20 feature file and change Destinatario into persona fisica

### CODE REVIEW

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java [hunk 1]
```diff
@@ -492,14 +492,14 @@ public void nellaSectionDestinatarioCliccareSuAggiungiDestinatario() {
         destinatarioPASection.selezionareAggiungiDestinatarioButton();
     }
 
-    @And("^Nella section Destinatario inserire i dati del destinatari aggiuntivi  per (.*)$")
-    public void nellaSectionDestinatarioInserireIDatiDelDestinatariAggiuntiviPerNumeroDestinatari(String nDestinatari) {
-        logger.info("Si cerca di aggiungere" + nDestinatari + " destinatari");
+    @And("^Nella section Destinatario inserire i dati delle persone fisiche aggiuntive per (.*)$")
+    public void nellaSectionDestinatarioInserireIDatiDelDestinatariAggiuntiviPerNumeroDestinatari(String npersoneFisiche) {
+        logger.info("Si cerca di aggiungere" + npersoneFisiche + " persone Fisiche");
```
Comment (leleOFA) [inline src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java:497]: change into "nPersoneFisiche"

Comment (leleOFA) [inline src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java:497]: as before

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java [hunk 2]
```diff
@@ -492,14 +492,14 @@ public void nellaSectionDestinatarioCliccareSuAggiungiDestinatario() {
...
         DataPopulation dataPopulation = new DataPopulation();
-        this.personaFisica = dataPopulation.readDataPopulation("destinatari.yaml");
+        this.personeFisiche = dataPopulation.readDataPopulation("personeFisiche.yaml");
         int nDestinatariInt = 1;
-        if (isNumeric(nDestinatari)) {
-            nDestinatariInt = Integer.parseInt(nDestinatari) - 1;
+        if (isNumeric(npersoneFisiche)) {
```
Comment (leleOFA) [inline src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java:501]: as befor

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java [hunk 3]
```diff
@@ -492,14 +492,14 @@ public void nellaSectionDestinatarioCliccareSuAggiungiDestinatario() {
...
+            nDestinatariInt = Integer.parseInt(npersoneFisiche) - 1;
```
Comment (leleOFA) [inline src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java]: as before

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java (hunk 1 above)
Comment (delfinoroNTTDATA) [reply src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java:497]: done

Comment (delfinoroNTTDATA) [reply src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java:497]: done

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java (hunk 2 above)
Comment (delfinoroNTTDATA) [reply src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java:501]: done

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java (hunk 3 above)
Comment (delfinoroNTTDATA) [reply src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java]: done

**Review by leleOFA — APPROVED** [review]

**Pull request merged**